src/assets/colors.js

```diff
@@ -1,11 +1,18 @@
 export default {
   accent: 'rgb(var(--etvas-accent-color))',
-  brand: 'rgb(var(--etvas-brand-color))',
+  brand: process.env.STORYBOOK_PRESENT
+    ? '#0040E3'
+    : 'rgb(var(--etvas-brand-color))',
+  brandLight: process.env.STORYBOOK_PRESENT
+    ? '#5585FF'
+    : 'rgb(var(--etvas-brand-color-light))',
```

Comment:
Please don't do this. We use exactly the same code for storybook and our portals. Using this `if` seems like you're trying to trick me.

Reply:
Use a default theme for storyboard

Reply:
I think I made it work